doc/tutorial/sequence_file/sequence_file_solution4.cpp

```diff
@@ -67,5 +74,7 @@ int main()
     {
         fout.push_back(rec);
     }
-}
+SEQAN3_DOXYGEN_ONLY(})
```

Comment:
Is this ok?
It does exactly what I want without touching the actual documentation.
Otherwise we would have to split it into two snippets...